partcad/src/partcad/project_factory_git.py

```
@@ -86,82 +113,113 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
+                            # Import the default branch
+                            if user_config.force_update or (now - os.path.getmtime(guard_path) > 24 * 3600):
+                                repo = Repo(cache_path)
+                                origin = repo.remote("origin")
+                                before = repo.active_branch.commit
+
+                                # If there is more than 1 remote branch, we have to
+                                # explicitly specify the branch to pull.
+                                remote_head = origin.refs.HEAD
+                                branch_name = remote_head.reference.name
+                                short_branch_name = branch_name[branch_name.find("/") + 1 :]
+                                pc_logging.debug("Refreshing the GIT branch: %s" % short_branch_name)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)